api/data_refinery_api/views.py

```diff
@@ -888,6 +890,7 @@ class Stats(APIView):
         description="Specify a range from which to calculate the possible options",
         enum=('day', 'week', 'month', 'year',)
     )])
+    @method_decorator(cache_page(60*5))  # Cache the stats view for five minutes
```

Comment:
Is it possible to use `@cache_page` directly here?

Reply:
I tried that first, but I think it's `Python`'s fault. `@method_decorator` wraps a function decorator to work with methods too. Without it, `@cache_page` tries to look at `self` instead of the `request` parameter.